EIPS/eip-7983.md

```diff
@@ -0,0 +1,64 @@
+---
+eip: 7983
+title: Transaction Gas Limit Cap at 16,777,216 Gas
```

Comment:
Suggestion to cap this at `2^24-1` such that it fits in a `bytes3` (so (16,777,215)

Reply:
I mentioned it on some ACD, when we were discussing 7825 too. By the time there wasn't much appetite but my feeling is, there's no way around.

Reply:
16,777,216 is nice because it makes it easier to subdivide things , potentially simplifying downstream engineering.